### DISCUSSION TITLE
I don't receive the mojo

### DISCUSSION THREAD

**Discussion by Ferillo** [Chia-Network/chia-blockchain#8144] [Support]
Hello,
I putted my chia wallet in https://faucet.chia.net/ for the mojo but I don't receive nothing.
My chia address is xch1c887zmxr7czsfam6wr9zp8jncehyv0ktsnk73vwwpnsyt42qv5ls0yx9pc
Can you help me?
Regards.

**Comment by BrandtH22** [comment]
@Ferillo , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it can take some time for the chia client to sync, make sure that the icon(s) in the top right corner are green and show synced. If this is the case and the funds are still not visible please reopen this ticket or let us know on discord.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
